modules/AlphaWalletFoundation/AlphaWalletFoundation/Transfer/eip1559/SuggestEip1559.swift

```diff
@@ -121,24 +121,24 @@ actor SuggestEip1559 {
 
         print("[GasEstimator] order: \(order)")
 
-        var maxBaseFee: Decimal = .zero
+        var largestBf: Decimal = .zero
```

Comment:
When I read this diff, I realised that maybe this is actually the same variable and your change was how it was supposed to work… even the old name makes sense now and why I wrote a comment about why the variable was unused (but forgot to post) 😆

Reply:
@oa-s Do you want to rename it back but otherwise include your changes related to this variable? Easier in case we compare to Kotlin next time